buckets/local/buckets.go

```diff
@@ -24,23 +23,16 @@ var (
 	ErrBucketExists = errors.New("bucket is already initialized")
 	// ErrThreadRequired indicates the operation requires a thread ID but none was given.
 	ErrThreadRequired = errors.New("thread ID is required")
-)
 
-// ConfConfig is used to generate new bucket configs.
```

Comment:
Ripping things out of this file and into `cmd`, where they can be used by other local libs.